### DISCUSSION TITLE
The definition of cco:Organism is too narrow

### DISCUSSION THREAD

**Discussion by neilotte** [CommonCoreOntology/CommonCoreOntologies#410] [Announcements]
The present definition is "An Object that is an Animal or Plant." This rules out bacteria, fungi, and hence is at odds with scientific standards.

**Comment by neilotte** [comment]
For additional context, see prior issue: https://github.com/CommonCoreOntology/CommonCoreOntologies/issues/30

**Comment by mark-jensen** [comment]
OBI [definition](http://purl.obolibrary.org/obo/OBI_0100026): "A material entity that is an individual living system, such as animal, plant, bacteria or virus, that is capable of replicating or reproducing, growth and maintenance in the right environment. An organism may be unicellular or made up, like humans, of many billions of cells divided into specialized tissues and organs."

This is the most reused class by other ontologies within Bioportal. The definition doesn't quite follow best practices. Notably it doesn't restrict organisms to being objects. I am ok with that. 

We can redefine along these lines: "A material entity that is an individual living system, capable of replicating or reproducing, growth and maintenance in the right environment.

Examples: Animal, Plant, Bacteria, Virus.
Elucidation:  An organism may be unicellular or made up, like humans, of many billions of cells divided into specialized tissues and organs.
source: http://purl.obolibrary.org/obo/OBI_0100026

Interesting to note that there is comment on the OBI term that says it's a placeholder. Dated 2009 :/

**Comment by johnbeve** [comment]
[I've objected](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0285093) to the OBI definition in print because the second sentence suggests viruses are cellular, which is false. 

The proposed definition takes a stand on viruses being alive, which is a contentious issue we should be neutral on; also the elucidation suggests viruses are cellular.

**Comment by mark-jensen** [comment]
@johnbeve would removing 'virus' form the list of examples address your concern?

Or do we want a def for 'organism' to include viruses but avoid use of 'alive'?

**Comment by johnbeve** [comment]
@mark-jensen I'm in favor of removing 'virus' from the scope; there are other agents viruses can be grouped with, e.g. viroids, satellites, prions, etc. and in general I think 'organism' is associated with 'living' or 'metabolizing' entities. [VIDO](https://bioportal.bioontology.org/ontologies/VIDO) provides such a classification. 

I  think we should avoid saying 'living' in the definition too, since including it in the definition and _not_ having virus as a subclass would _also_ suggest we're taking a stand. Better, I think, to remain neutral over what counts as 'living' since there are thorny issues around its use scientists need to work out.

**Comment by jonathanvajda** [comment]
We should also avoid "such as [example 1], [example 2]" locutions in definitions. Mark's recommendation to move them to examples of usage is fitting.

**Comment by alanruttenberg** [comment]
The OBI definition is marked as placeholder because it was thought it should be in some other domain ontology.
@johnbeve the second sentence should be understood  as a comment rather than definitional.
Having a definition of organism that deviates from OBI's, by excluding viruses is a bad idea, IMO. They are subject matter experts, not us.

**Comment by swartik** [comment]
@jonathanvajda I think the problem may be broader than that. Consider the definition of Act of Mass Media Communication:

    An Act of Communication intended to reach a large audience using a medium such as
    the internet, television, radio, newspaper, and magazine.

The object of "such as" refers to the medium, not the act. You wouldn't want to end the definition with "medium". How would you rewrite it? I can think of two approaches:

1. Add to CCO a class denoting a medium capable of handling acts of mass media communications, and rewrite the definition to use the name of that class.
2. End the definition with "audience", and put the rest in an elucidation annotation.

Your thoughts?

Most of the definitions that use "such as" are structured this way.

**Comment by cameronmore** [comment]
@swartik I am currently working on a proposal to remove the artifact use from the communication hierarchy. There is an act of communication and separately an act of artifact use. The way it's modeled right now, President Nixon's famous "I am not a crook" is _either_ an act of communication by mass media _or_ an act of denial. Under certain circumstances, maybe this is permissible (for an instance to be rdf:type two different communication classes) in some cases, but ultimately should be changed.
